### PR TITLE
fr: removing references to Rust (as in... "iron oxide", "rouille" in French)

### PR DESCRIPTION
**Pull request by yohcop**
Plus a few more things I saw on the way.

Related to French translation #323

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -1344,9 +1335,8 @@ msgstr ""
 "discussions/100) !"
 
 #: src/running-the-course/course-structure.md:5
-#, fuzzy
 msgid "Rust Fundamentals"
-msgstr "Binaires de rouille"
```
Comment (sakex) [inline po/fr.po:1349]: Excellent

File: po/fr.po [hunk 2]
```diff
@@ -3447,7 +3436,7 @@ msgstr ""
 #: src/basic-syntax/string-slices.md:20
 #, fuzzy
 msgid "Rust terminology:"
-msgstr "Terminologie de la rouille :"
+msgstr "Terminologie de la Rust :"
```
Comment (sakex) [inline po/fr.po:3439]:
```suggestion
msgstr "Terminologie de Rust :"
```

File: po/fr.po [hunk 3]
```diff
@@ -2792,14 +2782,13 @@ msgstr ""
...
-#, fuzzy
 msgid ""
 "The Rust standard library is small compared to languages like Java, Python, "
 "and Go. Rust does not come with several things you might consider standard "
 "and essential:"
 msgstr ""
-"La bibliothèque standard Rust est petite comparée à des langages comme Java, "
-"Python, et aller. La rouille ne vient pas avec plusieurs choses que vous "
+"La bibliothèque standard de Rust est petite comparée à des langages comme Java, "
+"Python, et Go. Rust n'est pas distribue avec plusieures choses que vous "
```
Comment (sakex) [inline po/fr.po:2791]:
```suggestion
"Python, et Go. Rust n'est pas distribué avec certains éléments que vous "
```

File: po/fr.po [hunk 4]
```diff
@@ -5109,13 +5098,12 @@ msgstr ""
...
 
 #: src/enums/sizes.md:3
-#, fuzzy
 msgid ""
 "Rust enums are packed tightly, taking constraints due to alignment into "
 "account:"
 msgstr ""
-"Les énumérations de rouille sont emballées étroitement, en tenant compte des "
-"contraintes dues à l'alignement :"
+"Les énumérations de Rust sont emballés étroitement, en tenant compte des "
```
Comment (sakex) [inline po/fr.po:5105]:
```suggestion
"Les énumérations de Rust sont emballées étroitement, en tenant compte des "
```

File: po/fr.po [hunk 5]
```diff
@@ -5370,13 +5358,12 @@ msgstr ""
...
 #: src/control-flow/while-let-expressions.md:21
 #: src/control-flow/match-expressions.md:23
-#, fuzzy
 msgid ""
 "See [pattern matching](../pattern-matching.md) for more details on patterns "
 "in Rust."
 msgstr ""
-"Voir [pattern matching](../pattern-matching.md) pour plus de détails sur les "
-"modèles dans Rouiller."
+"Voir [Filtrage par motif](../pattern-matching.md) pour plus de détails sur les "
```
Comment (sakex) [inline po/fr.po:5365]: Est-ce que ce lien marche vraiment?

File: po/fr.po [hunk 6]
```diff
@@ -13062,18 +13048,17 @@ msgstr ""
...
 "find a little corner of your code base where we can move some lines of code "
 "to Rust. The fewer dependencies and \"exotic\" types the better. Something "
 "that parses some raw bytes would be ideal."
 msgstr ""
 "Nous allons essayer d'appeler Rust depuis l'un de vos propres projets "
-"aujourd'hui. Alors essayez de trouvez un petit coin de votre base de code où "
-"nous pouvons déplacer quelques lignes de code vers Rouille. Moins il y a de "
-"dépendances et de types \"exotiques\", mieux c'est. Quelque chose que "
-"analyser quelques octets bruts serait idéal."
+"aujourd'hui. Alors essayez de trouvez un petit coin de votre code où "
```
Comment (sakex) [inline po/fr.po:13058]:
```suggestion
"aujourd'hui. Alors essayez de trouver un petit coin de votre code où "
```

File: po/fr.po [hunk 7]
```diff
@@ -13062,18 +13048,17 @@ msgstr ""
...
+"nous pouvons déplacer quelques lignes de code vers Rust. Le moins il y a de "
+"dépendances et de types \"exotiques\", le mieux c'est. Quelque chose qui "
+"analyse quelques octets bruts serait idéal."
```
Comment (sakex) [inline po/fr.po:13061]:
```suggestion
"analyse quelques octets de donnée brute serait idéal."
```

**Review by sakex — CHANGES_REQUESTED** [review]

**Comment by sakex** [comment]
Merci pour la review, les erreurs sont assez cocasses

File: po/fr.po (hunk 5 above)
Comment (yohcop) [reply po/fr.po:5365]: Oui, ca marche. Ce message est utilise dans src/control-flow/....md, donc ../pattern-matching.md est toujours valide.
C'est un peu un coup de chance ceci dit...

**Review by sakex — APPROVED** [review]

**Comment by yohcop** [comment]
Thanks for the review!

It looks like the ...@senges.ch email is missing the google cla (https://github.com/google/comprehensive-rust/pull/1214/checks?check_run_id=16863101887). I'm sure you did it with a different email already though.... :)

**Review by sakex — APPROVED** [review]

**Comment by sakex** [comment]
I wasn't aware of this. Thanks 😁

**Pull request merged**